src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -253,12 +253,11 @@ class HttpGoogleServicesDAO(
         //This can be used to auto-delete all objects next time the Google lifecycle manager runs (~every 24h).
         //More info: http://bit.ly/1WCYhhf and http://bit.ly/1Py6b6O
         val deleteEverythingRule = new Lifecycle.Rule()
-          .setAction(new Action().setType("delete"))
```

Comment:
If this has always been "delete" instead of "Delete", I can confirm that this never worked correctly.